docs/advanced/api.rst

```diff
@@ -14,12 +14,24 @@ Installation
 Fedora/CentOS
 -------------
 
-To allow remote access to the control agent REST API, and for agent connections,
+To enable Flocker control service.
```

Comment:
> perhaps explain a little about why this is necessary.

I'm going to say that this is outside the scope of this issue. This document isn't close to ready for an end-user to take and start using. When those documents are written, this can be expanded as necessary.